### PR TITLE
Add date and link for both Sustainability Week Milan events.

### PR DESCRIPTION
**Pull request by vallss**
Update Sustainability Week Milan events information with dates and links to meetup platform.

### CODE REVIEW

File: website/content/en/events/2024-cloud-native-sustainability-week.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | 3 | China | TBD | TBD | Sam Yuan
 | 4 | Oslo, Norway | Oct. 9 | [Link](https://www.meetup.com/gsf-oslo/events/303318869/) | Marta Paciorkowska, Kristina Devochko, [Green Software - Oslo](https://www.meetup.com/gsf-oslo)
 | 5 | Tokyo, Japan | Oct. 4 | [Link](https://community.cncf.io/events/details/cncf-cloud-native-community-japan-presents-cncf-cloud-native-sustainability-week-2024-local-meetup-tokyo/) | Sunyanan Choochotkeaw ([Cloud Native Community Japan](https://community.cncf.io/cloud-native-community-japan/))
-| 6 | Milan, Italy | TBD | TBD |  Michel Murabito, Valeria Salis, Ludovica Bonaldo
+| 6 | Milan, Italy | Sep. 30 & Oct. 7 | [Link Sep. 30](https://www.meetup.com/digital-platform-builders/events/303350215) - [Link Oct. 7](https://www.meetup.com/digital-platform-builders/events/303350378) |  Michel Murabito, Valeria Salis, Ludovica Bonaldo
```
Comment (guidemetothemoon) [inline website/content/en/events/2024-cloud-native-sustainability-week.md:36]: @vallss thanks for the PR! Can you please also update the same file for different languages? Here's an example from the earlier PR: https://github.com/cncf/tag-env-sustainability/pull/505/files

Comment (vallss) [reply website/content/en/events/2024-cloud-native-sustainability-week.md:36]: @guidemetothemoon thanks for reviewing it and of course, I'll do that later today!

Comment (vallss) [reply website/content/en/events/2024-cloud-native-sustainability-week.md:36]: @guidemetothemoon I noticed that other meetup information weren't updated so I updated all of them. Now we show the latest info for each meetup location in every language. Let me know if this is okay!

Comment (guidemetothemoon) [reply website/content/en/events/2024-cloud-native-sustainability-week.md:36]: @vallss looks great, thank you!

**Review by guidemetothemoon — APPROVED** [review]

**Pull request merged**